tiledb/sm/array/array.cc

```diff
@@ -200,6 +200,27 @@ Status Array::open(
     return LOG_STATUS(
         Status::ArrayError("Cannot open array; Array already open"));
 
+  std::string encryption_key_from_cfg;
+  if (!encryption_key) {
+    bool found = false;
+    encryption_key_from_cfg = config_.get("sm.encryption_key", &found);
+    assert(found);
+  }
```

Comment:
we generally leave a new-line after a `}` that is not immediately followed by another `}`.